blockchain/processing.py

```diff
@@ -255,6 +299,8 @@ def _execute_phase_1(self, config, current_block_id):
             # update status of approved transactions
             for tx in approved_transactions:
                 tx["header"]["status"] = "approved"
+                if tx["header"]["transaction_type"] == "TT_SUB_REQ":
```

Comment:
Did you see my other comment about letting transactions recieved via subscriptions also execute local smart contracts? I think using the event/notify system would work better for handling both cases, let me know if you want to discuss in person. I'm thinking we could extend the event system to use a http callback mechanism to invoke smart contracts running in separate containers etc...